lib/manageiq/rpm_build/build_uploader.rb

```diff
@@ -0,0 +1,59 @@
+module ManageIQ
+  module RPMBuild
+    class BuildUploader
+      attr_reader :release
+
+      def initialize(options = {})
+        @release = options[:release]
+      end
+
+      def upload
+        return unless OPTIONS.rpm_repository.s3_api.access_key && OPTIONS.rpm_repository.s3_api.secret_key
+
+        require 'digest'
+        puts "Uploading files..."
+        BUILD_DIR.glob(File.join('**', '*.rpm')).each do |file|
```

Comment:
I think there be a .sort here after the glob, so there is consistent ordering to the output